### PR TITLE
verification portal heading removed

### PR DESCRIPTION
**Pull request by bhavyaab**
## Description
- The top heading is removed and Login button is still available on the right side
- The gray background is removed and the background is completely white like designs
- VERIFICATION PORTAL heading is removed

## Test Plan
- yarn webpack 
- visit localhost/4000
- verify all changes

### CODE REVIEW

**Comment by karenmrose** [comment]
Nice! Would you mind also removing the banner at the top? The new designs have a completely white background so it's not needed.

banner at top

File: client/js/hah-app/header.js [hunk 1]
```diff
@@ -28,12 +26,7 @@ const styles = {
 		right: '0',
 		margin: 'auto',
 	},
-	authenticationButtonDiv: {
-		padding: '10px',
-		position: 'absolute',
-		top: '30px',
-		right: '30px'
-	},
+	authenticationButtonDiv: {},
```
Comment (karenmrose) [inline client/js/hah-app/header.js]: @bhavyaab If we're going to remove all the styles, we could also just remove this completely and remove the classNames for this style from above.

Comment (karenmrose) [reply client/js/hah-app/header.js]: It also looks like the logo style is not used anymore and can be removed...

**Comment by karenmrose** [comment]
Nice!! Thanks for doing the cleanup!! 👏 

Can you merge in master into this PR? I think the tests should pass then.

**Comment by karenmrose** [comment]
Hi @bhavyaab, this looks good, but we should get the tests to pass before merging in. To do that, here are the steps:
1. `git checkout master`
1. `git pull`
Note: you may need to revert the bundle file in order to pull from master and avoid conflicts. That's ok, we can rebuild it later
1. Once you've pulled from master, then run `git checkout verification_portal_heading`
1. And to merge: `git merge master`
1. Then rebuild the bundle by running `yarn webpack` 
1. Commit and push the changes.

Let me know if you have any questions!

**Comment by bhavyaab** [comment]
Thanks Karen,

**Pull request merged**